modules/client/src/connext.ts

```diff
@@ -55,7 +56,16 @@ import { falsy, notLessThanOrEqualTo, notPositive } from "./validation/bn";
  */
 
 export async function connect(opts: ClientOptions): Promise<ConnextInternal> {
-  const { logLevel, ethProviderUrl, mnemonic, natsClusterId, nodeUrl, natsToken, store } = opts;
+  const {
+    logLevel,
+    ethProviderUrl,
+    mnemonic,
+    natsClusterId,
+    nodeUrl,
+    natsToken,
+    webdisUrl,
```

Comment:
Remind me: This PR is the one that should obsolete anything webdis related right? So this URL should be removed before this PR is merged?

Reply:
Remove `webdisUrl`. We may need like `indraLockUrl` or similar though..

Reply:
Resolved.

> Remove webdisUrl. We may need like indraLockUrl or similar though..

We don't even, we just use the messaging service that the client already has.